client/README.md

```diff
@@ -11,8 +11,13 @@ This extension provides a Yseop Markup Language (YML) basic support for VS Code.
     * `granule`: Text granule
     * `instance`: Object instance
     * `textu`: Textualization attribute
+![Snippet for textualization Gif.](images/textualization.gif "Snippet for textualization.")
```

Comment:
Regarding the gif,
* I would have stopped before “number”, but the file's size if lower than I thought.
* Oh, and it's kinda weird that in the code you show there are three spaces before field names.
* And I would have removed about 150 (or even 250) pixels at the top and 180 at the bottom. And 300 pixels on the right. Don't forget this might end up being displayed in the “Extensions” panel of VSCode, which can be rather narrow (and cluttered) depending on what's opened alongside (and inside) it.

Reply:
* I wanted to show every options
* I don't “show” the spaces. I just click where I wanna start..
* Ok to remove pixels on top, but the right and the bottom's ones are usefull to display what happens in the suggestion list.

Reply:
* Yeah but you could clean the surrounding code before doing anything (regarding spurious spaces).
* I know the suggestions pop up and take lots of space, but still, you kept quite a lot of it. The thing that goes the farther to the right is what is displayed right before you trigger the textualization snippet, and I based my horizontal calculations on that, so… 300 pixels. `*checking*` Well, 260, at least.
![image](https://user-images.githubusercontent.com/36635838/41229957-4223257c-6d7e-11e8-88fc-77cf349ba7b0.png)

Reply:
ImageMagick → `convert` or `mogrify` → `crop` + https://www.imagemagick.org/Magick++/Geometry.html

Reply:
Omg omg

```
$ gifsicle --crop 0,250-895,690 --output new.gif textualization.gif
```